test/byContract/registries/resources.test.ts

```diff
@@ -0,0 +1,281 @@
+    PublicResolver,
+    PublicResolver__factory,
+    ReverseRegistrar__factory,
+    Workerpool,
+    WorkerpoolRegistry,
+    WorkerpoolRegistry__factory,
+    Workerpool__factory,
+} from '../../../typechain';
+import { MULTIADDR_BYTES } from '../../../utils/constants';
+import { getIexecAccounts } from '../../../utils/poco-tools';
+
+describe('Resources', () => {
+    let proxyAddress: string;
+    let iexecPoco: IexecInterfaceNative;
+    let [appProvider, datasetProvider, scheduler, anyone]: SignerWithAddress[] = [];
+
+    let ensRegistry: ENSRegistry;
+    let reverseResolver: PublicResolver;
+    let appRegistry: AppRegistry;
+    let datasetRegistry: DatasetRegistry;
+    let workerpoolRegistry: WorkerpoolRegistry;
+
+    let app: App;
+    let dataset: Dataset;
+    let workerpool: Workerpool;
+
+    beforeEach(async () => {
+        proxyAddress = await loadHardhatFixtureDeployment();
+        await loadFixture(initFixture);
+    });
+
+    async function initFixture() {
+        ({ appProvider, datasetProvider, scheduler, anyone } = await getIexecAccounts());
+        const ensRegistryAddress = (await deployments.get('ENSRegistry')).address;
+
+        iexecPoco = IexecInterfaceNative__factory.connect(proxyAddress, anyone);
+
+        appRegistry = AppRegistry__factory.connect(await iexecPoco.appregistry(), anyone);
+        datasetRegistry = DatasetRegistry__factory.connect(
+            await iexecPoco.datasetregistry(),
+            anyone,
+        );
+        workerpoolRegistry = WorkerpoolRegistry__factory.connect(
+            await iexecPoco.workerpoolregistry(),
+            anyone,
+        );
+
+        ensRegistry = ENSRegistry__factory.connect(ensRegistryAddress, anyone);
+        const reverseRootNameHash = ethers.utils.namehash('addr.reverse');
+        const reverseRegistrarAddress = await ensRegistry.owner(reverseRootNameHash);
+        const reverseResolverAddress = await ReverseRegistrar__factory.connect(
+            reverseRegistrarAddress,
+            anyone,
+        ).defaultResolver();
+        reverseResolver = PublicResolver__factory.connect(reverseResolverAddress, anyone);
+    }
+
+    describe('App', () => {
+        const createAppArgs = [
+            `App`,
+            'DOCKER',
+            MULTIADDR_BYTES,
+            ethers.utils.id(`Content of my app`),
+            '0x1234',
+        ] as [string, string, BytesLike, BytesLike, BytesLike];
+        beforeEach(async () => {
+            const appAddress = await appRegistry.callStatic.createApp(
+                appProvider.address,
+                ...createAppArgs,
+            );
+            await appRegistry
+                .createApp(appProvider.address, ...createAppArgs)
+                .then((tx) => tx.wait());
+            app = App__factory.connect(appAddress, anyone);
+        });
+
+        describe('creation and initialization', () => {
+            it('Should create an app and verify its details', async () => {
```

Comment:
```suggestion
            it('Should read initialized app details', async () => {
```
we don't create here we just read values and check them.

Reply:
Done [here](https://github.com/iExecBlockchainComputing/PoCo/pull/125/commits/0224386bfc4577c7d04e7087d153741716ba4690)